config/locales/error_messages.en.yml

```diff
@@ -701,6 +701,10 @@ representation_order:
       long: "Enter a valid MAAT reference for the #{representation_order} of the #{defendant}"
       short: Enter a valid MAAT reference
       api: "Enter a valid MAAT reference for the representation order of the defendant"
+    unique:
+      long: "Please ensure that MAAT references are unique for each defendant"
```

Comment:
just a content thing. A content person reviewed our error messages in the distant past and told use command-like sentences should be used, no pleases etc. So something like 'The MAAT references for the defendants are not unique'